source/artwork_import/vectorization.rst

```diff
@@ -1,3 +1,45 @@
-########################
-    Vectorization
-########################
+Vectorization of Bitmaps
+==============================================
+
+Synfig Studio comes with a feature of Vectorization of Bitmaps. In this tutorial, we will learn what is Vectorization feature and how to use it. :)
+
+What is Vectorization of Bitmaps?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+There are two common ways to represent a two-dimensional image:
+
+    * Raster Graphics or Bitmap Graphics
+    * Vector Graphics
+
+In Bitmap Graphics images are stored as a two-dimensional grid of pixels. This has some problems for example if a bitmap image is scaled it becomes blurry and pixelated.
+
+We use Vector Graphics to solve this problem. For converting Bitmap image to Vector Image, Synfig uses vectorization algorithm from `OpenToonz <https://opentoonz.github.io/e/>`_
+
+Algorithm
```

Comment:
I suggest to remove "Algorithm" section, as this information is not intended for user.